src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/HomeSteps/HomeSteps.java

```diff
@@ -0,0 +1,26 @@
+package org.umssdiplo.automationv01.stepdefinitionproject.HomeSteps;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import org.umssdiplo.automationv01.core.managepage.Home.SSIAHome;
+
+/**
+ * @Author: Lizeth Salazar
+ */
+public class HomeSteps {
+
+    @Given("SSI-A home page is loaded")
+    public void ssiAHomePageIsLoaded() throws Throwable {
+        SSIAHome.load();
+    }
+
+    @And("Clicking on Personnel menu")
+    public void clickPersonnelMenu() throws Throwable{
+        SSIAHome.clickOnPersonnelMenu();
```

Comment:
maybe as iti s the same object in this method you should use:
SSIAHome.clickOnPersonnelMenu();
employee = SSIAHome.clickOnEmployeeMenu();